content/2016/09/30/cssconf.rst

```diff
@@ -0,0 +1,640 @@
+public: yes
+author: miriam
+tags: [conference, front-end]
+image:
+  - src: 'cssconf-2016/ns.jpg'
+summary: |
+  It was a pleasure speaking at
+  CSSConf 2016 in Boston!
+  Here are some of my notes from the event.
+
+
+CSSConf 2016 Recap
+==================
+
+It was a pleasure speaking at
+CSSConf 2016 in Boston!
+Thanks to `Nicole Sullivan`_ for founding the series,
+`Bocoup`_ for hosting the event,
+(OddBird alum) `Claudina Sarahe`_ for her amazing emcee-ing,
```

Comment:
I feel like (OddBird alum) should go after her name, at first I thought you were talking about Nicole Sullivan.